src/styles/base.js

```diff
@@ -109,13 +111,15 @@ export const inputColor = '#A0A7B4';
 export const inputBorderRadius = '1px';
 export const inputPlaceholderColor = '#6A7485';
 export const inputPlaceholderFontWeight = 400;
-
+export const inputBoxShadow = 'none';
+export const inputBoxShadowActive = 'none';
 export const secondaryInputBgd = '#242730';
 export const secondaryInputBgdHover = '#3A414C';
 export const secondaryInputBgdActive = '#3A414C';
 export const secondaryInputColor = '#A0A7B4';
 export const secondaryInputBorderColor = '#242730';
 export const secondaryInputBorderActiveColor = '#D3D8E0';
+export const dropdownSelectHeight = '30px';
```

Comment:
make this `30` and in styled componenets use `props => props.theme. dropdownSelectHeight`px

Reply:
Done